diagnostics/model_accuracy/verify_inference_model.py

```diff
@@ -81,31 +58,25 @@ def run_inference(model_name, models_dir):
                     os.chdir(models_dir + "research/slim")
                 command_executor("export PYTHONPATH=$PYTHONPATH:`pwd`:`pwd`")
                 command_executor('git apply ' + pwd +
-                                 '/image_recognition.patch')
+                                 '/iage_recognition.patch')
```

Comment:
```suggestion
                                 '/image_recognition.patch')
```

Reply:
Thanks @sindhu-nervana, good catch, I will change it.